src/emulator/host/src/host.cpp

```diff
@@ -58,15 +65,27 @@ static Address resolve_export(KernelState &kernel, uint32_t nid) {
     return export_address->second;
 }
 
+void log_import_call(char emulation_level, uint32_t nid, SceUID thread_id, const std::unordered_set<uint32_t> &nid_blacklist) {
```

Comment:
```suggestion
static void log_import_call(const char emulation_level, const uint32_t nid, const SceUID thread_id, const std::unordered_set<uint32_t> &nid_blacklist) {
```

Reply:
we usually don't make fundamental value types `const` but yea I'll make it `static`

Reply:
TIL 😲